docs/Drawing.md

```diff
@@ -0,0 +1,189 @@
+# Drawing #
+
+The `fpdf.drawing` module provides an API for composing paths out of an
+arbitrary sequence of straight lines and curves. This allows fairly low-level
+control over the graphics primitives that PDF provides, giving the user the
+ability to draw pretty much any vector shape on the page.
+
+The drawing API makes use of features (notably transparency and blending modes)
+that were introduced in PDF 1.4. Therefore, use of the features of this module
+will automatically set the output version to 1.4 (the default is 1.3).
+
+## Getting Started
+
+The easiest way to add a drawing to the document is via `fpdf.FPDF.new_path`.
+This is a context manager that takes care of serializing the path to the
+document once the context is exited.
+
+Drawings follow the fpdf convention that the origin (that is, coordinate(0, 0)),
+is at the top-left corner of the page. The numbers specified to the various
+path commands are interpreted in the document units.
+
+```python
+import fpdf
+
+pdf = fpdf.FPDF(unit='mm', format=(10, 10))
+pdf.add_page()
+
+with pdf.new_path() as path:
+    path.move_to(2, 2)
+    path.line_to(8, 8)
+    path.horizontal_line_relative(-6)
+    path.line_relative(6, -6)
+    path.close()
+
+pdf.output("drawing-demo.pdf")
+```
+
+This example draws an hourglass shape centered on the page.
```

Comment:
Do you think I should add images of what the output looks like to the documentation? I thought about adding some, but it would involve committing several images to the documentation directory, so I decided not to do it at the time. I haven't checked if other documentation pages include images

Reply:
It's a good idea.
You can can some images in those docs if you want, I have nothing against that. Just try to use optimized JPEGs or webp files.
An alternative can be to provide links to generated example PDFs (the ones in the `test/` folder) as examples: they usually can be immediately viewed by users in their browser. They just won't appear as a "direct visual preview" in the documentation page